### DISCUSSION TITLE
Using Mixcr for hybridization capture data?

### DISCUSSION THREAD

**Discussion by lmcbride053** [milaboratory/mixcr#1399] [Q&A] [answered]
I have tried multiple different presets for processing hybrid-capture sequences in MiXCR using the analyze command and have not been able to retrieve the expected results. What sort of command/preset/features would you recommend using to process hybrid-capture data? I have seen some studies that have done it successfully.

**Comment by mizraelson** [comment]
Hi, can you describe in more detail the structure of your library?

**Reply by lmcbride053** [reply]
We used the Twist NGS Library Preparation Kits for our library preparation. Does that answer your question?

**Reply by mizraelson** [reply, marked as answer]
Have you tried the command bellow:

```
mixcr analyze exome-seq \
    --species hsa \
      input_R1.fastq.gz \
      input_R2.fastq.gz \
      result
```

What command did you try and what was unexpected about the results?

Can you share the studies you have mentioned?

Sincerely,
Mark
